NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Tyler branch view all posts

# What changed?

Added the ability to view all posts.  Addresses Trello Ticket 3, "View All Posts"

# How to test?

1. Add and merge whatever you're working on now
2. git checkout main
3. git fetch --all
4. git checkout tyler-branch-viewAllPosts
5. If you haven't already, create and seed your databases using the SQL information.
6. Run your API out of visual studio (the purple one)
7. Run npm start out of your terminal to start the app
8. Log into the site if you haven't already.  foo@bar.com works as an email, skip the password
9. Click on "Posts" on the navbar
10. You should see a list of posts.  None of the buttons work yet, but you'll see all posts there.
11. Congratulations, you have viewed all the posts.
